CuratedContent/CMakeTutorialAndReferenceResources.md

```diff
@@ -0,0 +1,87 @@
+This book covers every basic CMake feature in great detail.
+It explains modern CMake idioms and best practices.
+Also, it covers older CMake commands and idioms to avoid and why.
+This book seems to answer the "why" and big picture questions for CMake in a way that no other material has matched.
+It also provides really critical low-level details and tips that are absent in even the official CMake reference documentation.
+This book is self published by the author in a downloadable PDF format with new editions coming out after each new CMake release (i.e. three new editions each year).
+The current edition of the book costs $30 which includes 9 updates for new editions of the book as they come out.
+(Therefore, $30 would cover every new edition over a 3 year period which includes 9 new CMake releases or 6 years if only downloading every other updated edition across 18 CMake releases.)
+The author is not currently a Kitware employee but is a significant contributor to CMake and collaborates closely with Kitware staff.
+The author starts evaluating and updating the book at the beginning of each new CMake release cycle (i.e. right after the first release candidate `vX.Y.0-rc1` is tagged).
+Through this process, the author often provides feedback to Kitware and help addressing issues with new CMake features and changes to existing CMake functionality which feeds back into bug fixes and better CMake reference and release notes documentation.
+
+Another important source of CMake documentation is Kitware's official online ***CMake Reference Documentation***.
+This documentation is produced and versioned for every CMake release and contains a wealth of information including:
+
+* Reference material for the entire CMake language including for every supported command, property, variable and related topic
+* Documentation for the large collection of standard `*.cmake` modules that are installed along with CMake
+* Targeted guides including:
+  * CMake Tutorial (for developers contributing to projects that use CMake)
+  * User interaction Guide (for users trying to configure, build, install and test projects that use CMake)
+* Release notes for each version of CMake
+* Command-line documentation for the `cmake`, `ctest`, and `cpack` executables
+
+As a developer contributing to a project using CMake (i.e. editing the project's `CMakeLists.txt` files), when consulting the official reference documentation, it is advisable to only view documentation for the minimum version of CMake supported by the project (i.e. for the version `X.Y` listed in the project's top-level `CMakeLists.txt` file calling `cmake_minimum_required(VERSION X.Y)`).
+For example, if minimum version was 3.17, then one would select that version in the drop-down box (taking you to https://cmake.org/cmake/help/v3.17/).  This helps to avoid accidentally using CMake features not present in the minimum CMake version supported by the project.
```

Comment:
```suggestion
For example, if minimum version is 3.17, then one would select that version in the drop-down box (taking you to https://cmake.org/cmake/help/v3.17/).  This helps to avoid accidentally using CMake features available only in newer versions.
```